core/java/com/google/instrumentation/stats/IntervalAggregationDescriptor.java

```diff
@@ -24,18 +24,47 @@
  */
 public final class IntervalAggregationDescriptor {
   /**
-   * Constructs a new {@link IntervalAggregationDescriptor}. The given {@code intervalSizes} must
-   * have at least one entry.
+   * Constructs a new {@link IntervalAggregationDescriptor}.
+   *
+   * <p> The number of internal sub-intervals to use when collecting stats for each interval. The
+   * max error in interval measurements will be approximately 1/getNumSubIntervals()
+   * (although in practice, this will only be approached in the presence of very large and bursty
+   * workload changes), and underlying memory usage will be roughly proportional to the value of
+   * this field. Must be in the range [2, 20]. A value of 5 will be used if this is unspecified.
```

Comment:
since this isn't nullable, can it be unspecified? I think the default is covered in the overloads